libraries/lib-ipc/IPCChannel.h

```diff
@@ -13,6 +13,9 @@
 #pragma once
 
 #include <cstddef>
+#ifndef _WIN32
+#include <sys/select.h>
```

Comment:
`IPCChannel.h` is a public interface file, please move this include to `internal/ipc-types.h`